pallets/subtensor/src/registration.rs

```diff
@@ -528,18 +528,14 @@ impl<T: Config> Pallet<T> {
         hash_as_vec
     }
 
-    #[allow(clippy::indexing_slicing)]
     pub fn hash_block_and_hotkey(block_hash_bytes: &[u8; 32], hotkey: &T::AccountId) -> H256 {
-        // Get the public key from the account id.
-        let hotkey_pubkey: MultiAddress<T::AccountId, ()> = MultiAddress::Id(hotkey.clone());
-        let binding = hotkey_pubkey.encode();
-        // Skip extra 0th byte.
-        let hotkey_bytes: &[u8] = binding[1..].as_ref();
```

Comment:
This one I'm fairly certain but not 100% sure -- it's actually trying to skip the discriminant byte from the `MultiAddress` enum, right? If so, we can just encode the `hotkey` directly, as I've done in my PR.

Reply:
This seems right to me